flocx_market/api/offer_contract_relationship.py

```diff
@@ -6,34 +6,56 @@
 class OfferContractRelationship(Resource):
 
     @classmethod
-    def get(cls, marketplace_offer_id=None, contract_id=None):
-        if (contract_id is None) and (marketplace_offer_id is None):
+    def get(cls):
```

Comment:
I'm not terrible familiar with flask - should this be `def get(cls, offer_contract_relationship_id)`?

Reply:
since we are grabbing it from request.args.get(), then we don't need it in the method parameters @tzumainn

Reply:
Is there a reason *not* to make this parallel to the way the offer/bid/contract code is written though?